src/inference/dev_api/ie_system_conf.h

```diff
@@ -97,6 +97,13 @@ INFERENCE_ENGINE_API_CPP(bool) with_cpu_x86_avx512f();
  */
 INFERENCE_ENGINE_API_CPP(bool) with_cpu_x86_avx512_core();
 
+/**
+ * @brief      Checks whether CPU supports AVX 512 VNNI capability
+ * @ingroup    ie_dev_api_system_conf
+ * @return     `True` is AVX512F, AVX512BW, AVX512DQ, AVX512_VNNI instructions are available, `false` otherwise
+ */
+INFERENCE_ENGINE_API_CPP(bool) with_cpu_x86_avx512_core_vnni();
```

Comment:
@dmitry-gorokhov Maybe it already was before, but why do we extend developer API only in order to check CPU in tests? Maybe will be better to have a test utility for this purpose.

Reply:
With gapi preprocessing elimination most of the with_cpu_x86* utilities are used only in CPU tests. 
From that perspective we can follow two ways:
1. Keep with_cpu_x86* utility consistent and support all popular ISA configuration
2. Move all used by CPU tests inside CPU tests and leave here only functions used by other parts of the product.

Reply:
Discussed offline. And we will refactor this API in future